File: server/test/reader-test/src/component-test/java/com/richo/reader/test/pages/model/FeedWithoutItem.java
package com.richo.reader.test.pages.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public class FeedWithoutItem
{
	public final String id;
	public final String name;
	public final int numberOfAvailableItems;

	public FeedWithoutItem(@JsonProperty("id") final String id, @JsonProperty("name") final String name, @JsonProperty("numberOfAvailableItems") final int numberOfAvailableItems)
	{
		this.id = id;
		this.name = name;
		this.numberOfAvailableItems = numberOfAvailableItems;
	}
}

File: reader/server/core/spring/src/test/kotlin/com/richodemus/reader/reader/test/pages/model/FeedWithoutItem.kt
package com.richodemus.reader.reader.test.pages.model

import com.fasterxml.jackson.annotation.JsonProperty

class FeedWithoutItem(@param:JsonProperty("id") val id: String, @param:JsonProperty("name") val name: String, @param:JsonProperty("numberOfAvailableItems") val numberOfAvailableItems: Int)
